tests/integration/models/keys.spec.coffee

```diff
@@ -76,7 +76,9 @@ describe 'Key Model', ->
 
 			it 'should be rejected if the key id is invalid', ->
 				promise = resin.models.key.get(99999999999)
-				m.chai.expect(promise).to.be.rejectedWith('Request error: Internal Server Error')
+				m.chai.expect(promise).to.be.rejectedWith('Request error: value "99999999999" is out of range for type integer')
```

Comment:
Should we just test for `rejectedWith('Request error')`? Since it seems that the error historically tends to change.